qual_id/categories/games.py

```diff
@@ -0,0 +1,20 @@
+from ..category import Category
+
+
+class Games(Category):
+  def get_values(self):
+    return [
+        'basketball',
+        'cricket',
+        'baseball',
+        'hockey',
+        'chess',
+        'tennis',
+        'table-tennis',
```

Comment:
Would you mind removing 'table-tennis' or replacing it as it contains a '-' which could be confusing. 
I meant to write tests to cover this but I forgot so thanks for reminding me 🙂

Reply:
Hey @gabrielbarker, can't we use camelCase or use '_' instead of '-'?

Reply:
That's a good point. I'd prefer it if you remove it for now, but if do it in a single commit, then it'll be easy to revert in future if that's what we decide in future, thanks

Reply:
sure, I ll keep in mind. thanks